client/src/modules/Opportunities/components/Link/index.tsx

```diff
@@ -5,11 +5,9 @@ type Props = {
 };
 
 export const Link = ({ url, text, title }: Props) => {
-  return url ? (
+  return (
```

Comment:
this component wasn't used as link without href before and I don't think that it will be used this way, so I decided to make url mandatory and skip this check